sw/device/tests/sim_dv/all_escalation_resets_test.c

```diff
@@ -95,13 +90,13 @@ const volatile uint32_t nv_nmi_interrupt_counter = UINT32_MAX;
  * - bite after escalation reset, so we should not get timer reset.
  */
 enum {
-  kWdogBarkMicros = 10 * 100,         //   1 ms
-  kWdogBiteMicros = 15 * 100,         // 1.5 ms
-  kEscalationStartMicros = 40,        //  40 us
-  kEscalationPhase0Micros = 2 * 100,  // 200 us
-  kEscalationPhase1Micros = 2 * 100,  // 200 us
-  kEscalationPhase2Micros = 100,      // 100 us
-  kEscalationPhase3Micros = 100,      // 100 us
+  kWdogBarkMicros = 2000,          // 2 ms
```

Comment:
@matutem might need your help here to review.
This looks okay to me, but i'm not sure if extending some of the phases changes any assumptions you had.

Reply:
just a quick note: I was able to run the `chip_sw_all_escalation_resets` in `dvsim` and 59/60 passed (same result before these changes)

Reply:
I had to increase `kEscalationStartMicros` so the the regular ISR had enough time to write to the flash, and kind of scaled up the rest. running time of the test doesn't seem to be affected, still ~40 mins.

Reply:
ah sounds good. I think it should be fine... what this test is going for I think isn't really related to the length of the escalation. But it would be good for @matutem to confirm.